grapple/types/redirects.py

```diff
@@ -1,31 +1,52 @@
+import json
+
+from typing import Optional
+
 import graphene
 
-from django.conf import settings
 from wagtail.contrib.redirects.models import Redirect
+from wagtail.models import Page, Site
 
 from .pages import get_page_interface
 
 
 class RedirectType(graphene.ObjectType):
```

Comment:
**Issue:** Are we missing a `site` field? `wagtail.contrib.redirects.models` contains a `site` field, so why doesn't our `ObjectType` contain one?

As an API consumer calling a GraphQL API using wagtail-grapple, how would I filter the redirects for a specific site? Note, this last point should be considered on top of adding a `site` field - perhaps a query would help?

Reply:
Thanks, added a site field in 33561505891fe6ec7d6a5901f9301af1d26aaad5